.github/workflows/ci.yml

```
@@ -0,0 +1,34 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: CI
+
+on:
+  push:
+  
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [14.17.x, 16.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      if: ${{ !env.ACT }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - name: Use Node.js ${{ matrix.node-version }}
+      if: ${{ env.ACT }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
```

Action template supports binary logic, so "use cache if not env.ACT" can be implemented like this:
```suggestion
    - name: Use Node.js ${{ matrix.node-version }}
      uses: actions/setup-node@v2
      with:
        node-version: ${{ matrix.node-version }}
        cache: ${{ !env.ACT && 'npm' || '' }}
```

I haven't tested, but maybe even a ternary would be supported `${{ env.ACT ? '' : 'npm' }}`

Reference for logic in template:
https://github.com/actions/runner/issues/409

The reason empty string works is because `cache` input in an action defined here:
https://github.com/actions/setup-node/blob/9ced9a43a244f3ac94f13bfd896db8c8f30da67a/action.yml#L24-L25

Is checked for truthiness, rather than any explicit value, here:
https://github.com/actions/setup-node/blob/9ced9a43a244f3ac94f13bfd896db8c8f30da67a/src/main.ts#L48

---

Thank's! I didn't know that :)
An empty string will work, but I think that it will throw a warning during the execution of the pipeline. Take a look here https://github.com/rangle/radius-tracker/actions/runs/1945607661 Are we okay with that?

---

Eh, I'm fine with that. I can't even find where `disable-cache` is coming from.